ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -18,7 +18,8 @@
 
 public class ActionManager
 {
-
+	// this class no longer serves any useful purpose and will be removed in a future release
+	@Deprecated(forRemoval = true)
```

Comment:
This seems for a different pull request to me as I don't think it is necessary for the current change set to function?

Reply:
Yeah, it is indeed another PR. I will open a new JBTM to deprecate this class

Reply:
Done. New [JBTM](https://issues.redhat.com/browse/JBTM-3897) created. Thanks Tom